### ISSUE TITLE
Dropdown Component has no accessibility - NVDA

### ISSUE THREAD

**Issue by morane1** [primefaces/primeng#10825]
still not working on PrimeNg 12.2.1 

NVDA accessibility program cannot read the internal content of the dropdown, when you navigate with arrows it only informing on selected item.

similar issue 
https://github.com/primefaces/primeng/issues/9342

**Comment by yigitfindikli** [comment]
Hi @morane1,

All accessibility issues will be fixed in 2022 Q1. We're gonna publish accessibility scores as well. Thanks for your report.
